pkg/finder/finder_test.go

```diff
@@ -40,6 +42,24 @@ func Test_fsFinderExcludeDirs(t *testing.T) {
 	}
 }
 
+func Test_fsFinderExcludeFileTypes(t *testing.T) {
```

Comment:
I see you added this test but I don't see it ever executed in the test stage of the pipeline.  Please add a retest.

Reply:
https://github.com/Boeing/config-file-validator/actions/runs/6189865772/job/16888184708#step:4:76

I added tests in later commits, perhaps you just checked the wrong pipeline